content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -108,7 +108,7 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
 
 1. `kubernetes.io/legacy-unknown`:  has no guarantees for trust at all. Some distributions may honor these as client
   certs, but that behavior is not standard Kubernetes behavior.
-  This signerName can only be requested in CertificateSigningRequests created via the `certificates.k8s.io/v1beta1` API version.
+  This signerName can only be requested in CertificateSigningRequests created via the `certificates.k8s.io/v1` API version.
```

Comment:
The original text is wrong, but the revised text doesn't mention that `certificates.k8s.io/v1beta1` also works.

You could consider dropping the line now that the feature is GA.

Reply:
Thanks for the feedback @sftim 
I will remove the line.